deployment/secure_research_desktop/packages/packages-python.yaml

```diff
@@ -41,24 +38,19 @@ packages:
     Pillow:
     plotly:
     poetry:  # also used by installation scripts
-        "all": [">1.0.0"]  # increase solver flexibility
+        "all": [">=1.3.0,<1.4.0"]  # avoid known installation issue https://github.com/python-poetry/poetry/issues/7611
     prophet:
-    psycopg2:
     pydot:
-    pygrib:
     pylint:
     pymc:
     pyodbc:
     pyproj:
     pyshp:
-    pystan:
     pytest:
     PyYAML:
     regex:
     requests:
     safety:  # also used by installation scripts
-    scikit-image:
```

Comment:
Instead of removing `scikit-image` could you mark it as `uninstallable` for all Python versions and explain why?

I'm not sure what dependency problems this packages would have.